packages/frontend/src/pages/Others/ManagementReports/index.tsx

```diff
@@ -1,3 +1,48 @@
+import { Button, TextField } from '@mui/material';
+import { useState } from 'react';
+import { ContainerWrapper } from './index.styles';
+import FileUploadService from '../../../services/FileUploadService';
+
+const Form = () => {
+  const selectFile = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { files } = event.target;
+    const selectedFiles = files as FileList;
+    setFile(selectedFiles?.[0]);
+  };
+  const upload = () => {
+    if (!file) return;
+
+    FileUploadService.upload(nodeId, file).catch((err) => {
+      if (err.response && err.response.data && err.response.data.message) {
+      } else {
+        console.log('something went wrong!');
```

Comment:
Proper error handling? Or at least a TODO. Here and with this whole function. Should this be a function that's imported from a utils or something, as it's going to be needed on so many pages?